compass/landice/tests/ismip6_run_ais/set_up_experiment.py

```diff
@@ -0,0 +1,215 @@
+                forcing_basepath = os.path.join(forcing_basepath, 'tier2_experiments')
+            else:
+                forcing_basepath = os.path.join(forcing_basepath, 'tier1_experiments')
+        else:
+            forcing_basepath = os.path.join(forcing_basepath, 'tier1_experiments')
+
+        # Copy files we'll need from local paths specified in cfg file
+        if self.exp == 'hist':
+            shutil.copy(init_cond_path, self.work_dir)
+        shutil.copy(melt_params_path, self.work_dir)
+        shutil.copy(region_mask_path, self.work_dir)
+
+        # Find and copy correct forcing files
+        smb_search_path = os.path.join(forcing_basepath, exp_fcg, 'processed_SMB_*_smbNeg_over_bareland.nc')
+        fcgFileList = glob.glob(smb_search_path)
+        if len(fcgFileList) == 1:
+            smb_path = fcgFileList[0]
+            smb_fname = os.path.split(smb_path)[-1]
+            shutil.copy(smb_path, self.work_dir)
+        else:
+            sys.exit(f"ERROR: Did not find exactly one matching SMB file at {smb_search_path}: {fcgFileList}")
+
+        tf_search_path = os.path.join(forcing_basepath, exp_fcg, 'processed_TF_*.nc')
+        fcgFileList = glob.glob(tf_search_path)
+        if len(fcgFileList) == 1:
+            tf_path = fcgFileList[0]
+            tf_fname = os.path.split(tf_path)[-1]
+            shutil.copy(tf_path, self.work_dir)
+        else:
+            sys.exit(f"ERROR: Did not find exactly one matching TF file at {tf_search_path}: {fcgFileList}")
+
+        # Make stream modifications based on files that were determined above
+        stream_replacements = {
+                               'input_file_SMB_forcing': smb_fname,
+                               'input_file_TF_forcing': tf_fname
+                               }
+        if self.exp == 'hist':
+            stream_replacements['input_file_init_cond'] = init_cond_fname
+            stream_replacements['input_file_region_mask'] = region_mask_fname
+            stream_replacements['input_file_melt_params'] = melt_params_fname
+        else:
+            stream_replacements['input_file_init_cond'] = 'USE_RESTART_FILE_INSTEAD'
+            stream_replacements['input_file_region_mask'] = 'USE_RESTART_FILE_INSTEAD'
+            stream_replacements['input_file_melt_params'] = 'USE_RESTART_FILE_INSTEAD'
+        if self.exp == 'hist' or self.exp == 'ctrlAE':
+            stream_replacements['forcing_interval'] = 'initial_only'
+        else:
+            stream_replacements['forcing_interval'] = '0001-00-00_00:00:00'
+
+        self.add_streams_file(
+            'compass.landice.tests.ismip6_run_ais', 'streams.landice.template',
+            out_name='streams.landice',
+            template_replacements=stream_replacements)
+
+        # Set up namelist and customize as needed
+        self.add_namelist_file(
+            'compass.landice.tests.ismip6_run_ais', 'namelist.landice',
+            out_name='namelist.landice')
+
+        if mesh_res == '04':
```

Comment:
```suggestion
        if mesh_res == 4:
```

Reply:
I think it would be better to have an `elif mesh_res == 8:` here as well and set teh corresponding options there (even if they happen to be the defaults) and then have:
````
else:
    raise ValueError(f'Unsupported mesh resolution {mesh_res}.')